server/db.js

```diff
@@ -0,0 +1,20 @@
+  
+require('dotenv').config();
+const { Pool } = require('pg')
+
+const user = process.env.PG_USERNAME || "postgres";
+const password = process.env.PG_PASSWORD || "postgres";
+const host = process.env.PG_HOST || "localhost";
+const dbName = process.env.PG_DB || "jobs"
+
+const pool = new Pool({
+  user: user,
+  host: host,
+  database: dbName,
+  password: password,
+  post: 5432
```

Comment:
Perhaps this should be set in environment too ⚔️ 😃 . Default value 5432, and load from env.

Reply:
Also... `post` typo, should be `port`